sap-xmake-build.sh

```diff
@@ -0,0 +1,119 @@
+#!/bin/bash -l
+
+# This build script is only applicable to Spark without Hadoop and Hive
+
+PACKAGE_BRANCH="alti-branch-0.5"
```

Comment:
should this be pointing to the `sap-branch-0.5.1-alti` which is a clone from `alti-branch-0.5`?